src/app/services/reducers/devxApi-reducers.ts

```diff
@@ -2,7 +2,11 @@ import { IAction } from '../../../types/action';
 import { DEVX_API_URL } from '../graph-constants';
 import { SET_DEVX_API_URL_SUCCESS } from '../redux-constants';
 
-export function devxApi(state: string = DEVX_API_URL, action: IAction): any {
+const initialState = {
+  baseUrl: DEVX_API_URL,
+  parameters: ''
+};
```

Comment:
Let's create a type `IDevxApi` for this. and save it in the types folder. So that we reference it like so 
```
const initialState: IDevxApi = {
 baseUrl: DEVX_API_URL,
 parameters: ''
}
```